<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LichtEinstellungen</class>
 <widget class="QDialog" name="LichtEinstellungen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QToolButton" name="lichtResetTimerToolButton_0">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>169</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="lichtResetTimerToolButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>219</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="lichtResetTimerToolButton_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>269</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="lichtResetTimerToolButton_6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>319</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="lichtResetTimerToolButton_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>219</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="lichtResetTimerToolButton_1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>169</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="lichtResetTimerToolButton_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>269</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="lichtResetTimerToolButton_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>319</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>129</y>
     <width>401</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QPushButton" name="pushButton_Apply">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ãœbernehmen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Cancel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Abbrechen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reset Timer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Beleuchtung - Einstellungen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>lichtResetTimerToolButton_0</zorder>
  <zorder>lichtResetTimerToolButton_2</zorder>
  <zorder>lichtResetTimerToolButton_4</zorder>
  <zorder>lichtResetTimerToolButton_6</zorder>
  <zorder>lichtResetTimerToolButton_3</zorder>
  <zorder>lichtResetTimerToolButton_1</zorder>
  <zorder>lichtResetTimerToolButton_5</zorder>
  <zorder>lichtResetTimerToolButton_7</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Apply</sender>
   <signal>clicked()</signal>
   <receiver>LichtEinstellungen</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>LichtEinstellungen</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>751</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
